packages/@webex/plugin-meetings/test/unit/spec/meetings/index.js

```diff
@@ -412,10 +412,8 @@ describe('plugin-meetings', () => {
           assert.exists(result);
           assert.instanceOf(result, NoiseReductionEffect);
           assert.containsAllKeys(result, ['audioContext', 'isEnabled', 'isReady', 'options']);
-          assert.deepEqual(result.options, {
-            authToken: 'fake_token',
-            audioContext: {},
-          });
+          assert.equal(result.options.authToken, 'fake_token');
+          assert.deepEqual(result.options.audioContext, {});
```

Comment:
suggestion: Since the next test will fail if the media effects library changes what is in the options (so that test will need fixing), we may as well do a full check of the options here also, adding in the missing items.

Reply:
another thing we could do is to just modify the next test to just check the individual options fields that we pass. It seems weird that we're verifying all the options when they actually belong to the effects lib and are controlled by that lib and they can keep changing them.